nextflow_schema.json

```diff
@@ -385,6 +385,10 @@
             "description": "",
             "default": "",
             "properties": {
+                "coassemble_group": {
+                    "type": "boolean",
+                    "description": "Co-assemble samples within one group, instead of assembling each sample separately."
```

Comment:
ah, no, if not using `input.tsv` but `"reads_R{1,2}.fastq.gz"`, all reads will end up in the same group (`0`), and thus can also be co-assembed.

Reply:
I will add this information to the docs, it might be a bit too much for `help`

Reply:
so by default, the group information is not used, neither for co-assembly nor for computing co-abundances ...

Reply:
That seems to be also a really important info.

Would it be better to make default for co-abundances to group? In case the input is without manifest.tsv, all samples have "group" 0 and it still would equal "all", correct?

For co-assembly, I am not sure whats the best default. If we choose here "coassemble_group=true", than without manifest.tsv, one assembly would be generated, correct? I am a little afraid that people (incl. me) will produce a neat manifest but forget to specify `--coassemble_group`...

Reply:
I agree regarding setting the default of 'binning_map_mode' to `group`

When setting the default of `coassemble_group` to `true`, I could try to add different groups for each sample when not using a `manifest.tsv`. Setting different defaults might be confusing.

Reply:
hm... having each sample in a different group is also not nice, since the assemblies will be named `groupX` then.

Reply:
Why not call group like the name of the sample right when parsing input instead of "0"?

e.g. https://github.com/skrakau/mag/blob/f773cb74863cc5e07bc1e681b5890d5e2f91a094/main.nf#L183
instead of
`.map { row -> [ row[0], 0, [ file(row[1][0], checkIfExists: true) ] ] }`
use
`.map { row -> [ row[0], row[0], [ file(row[1][0], checkIfExists: true) ] ] }`
might that work?

Reply:
true, anyway all samples require unique names in this case

Reply:
see below...